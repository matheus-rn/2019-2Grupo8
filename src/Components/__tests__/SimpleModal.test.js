import React from 'react';
import Index from '../SimpleModal/index';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Testing index component', () => {
    it('Test if index renders correctly', () =>{
        const tree = shallow(<Index/>);
        expect(toJson(tree)).toMatchSnapshot();
    });
});
