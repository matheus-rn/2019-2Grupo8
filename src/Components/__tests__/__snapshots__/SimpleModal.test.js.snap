// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing index component Test if index renders correctly 1`] = `
<Route>
  <Component />
</Route>
`;
